tests/st/harness/core/test_runner.py

```
@@ -423,62 +403,61 @@ def _load_kc(work_dir: Path):
 
     with concurrent.futures.ThreadPoolExecutor(max_workers=max_workers) as pool:
         all_futs: list = []
 
-        # Runtime builds (independent of test-case kernels)
-        for tc_platform, runtime_name in seen_runtimes:
-            all_futs.append(pool.submit(_build_runtime_task, tc_platform, runtime_name))
-
         # Per-test-case: each kernel and orchestration as independent tasks
         tc_kernel_futs: list[tuple[list, concurrent.futures.Future]] = []
-        for tc, compiler, runtime_name, kernels, orch_source in case_configs:
+        for tc, compiler, tc_platform, runtime_name, kernels, orch_source in case_configs:
```

Ruff (0.15.9)

[warning] Loop control variable `tc` not used within loop body

(B007)